project-base/app/tests/FrontendApiBundle/Functional/Cart/RemoveCartTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\FrontendApiBundle\Functional\Cart;
+
+use App\DataFixtures\Demo\ProductDataFixture;
+use App\Model\Product\Product;
+use Tests\FrontendApiBundle\Test\GraphQlTestCase;
+
+class RemoveCartTest extends GraphQlTestCase
+{
+    private Product $testingProduct;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->testingProduct = $this->getReference(ProductDataFixture::PRODUCT_PREFIX . 1, Product::class);
+    }
+
+    public function testCartIsRemoved(): void
+    {
+        $firstProductQuantity = 6;
+        $newlyCreatedCart = $this->addTestingProductToNewCart($firstProductQuantity);
+        $secondProduct = $this->getReference(ProductDataFixture::PRODUCT_PREFIX . 72, Product::class);
+        $secondProductQuantity = 3;
+
+        $this->getResponseContentForGql(__DIR__ . '/../_graphql/mutation/AddToCartMutation.graphql', [
+            'cartUuid' => $newlyCreatedCart['uuid'],
+            'productUuid' => $secondProduct->getUuid(),
+            'quantity' => $secondProductQuantity,
+        ]);
```

Comment:
Same as above, I do not think we need two products in the cart for purposes of this test